src/components/Search/Search.jsx

```diff
@@ -1,19 +1,24 @@
 import { Box, Typography } from "@mui/material";
-import { styleTitle } from "./Search.style";
-import Input from "../Input/Input";
-import DeliveryButton from "../DeliveryButton/DeliveryButton";
+import DeliveryButton from "./components/DeliveryButton/DeliveryButton";
+import Input from "./components/Input/Input";
 
 const Search = () => {
   return (
     <Box sx={{ display: "flex", mt: "3rem" }}>
-      <Box sx={styleTitle}>
+      <Box sx={{ display: "flex", flexGrow: 1 }}>
         <Typography
           color="initial"
           sx={{ display: "flex", fontSize: "30px", mr: "10px" }}
         >
           Delivery
         </Typography>
-        <Typography sx={{ color: "#00000060", fontSize: "30px" }}>
+        <Typography
+          sx={{
+            color: "#00000060",
+            fontSize: "30px",
+            fontWeight: 300,
+          }}
+        >
```

Comment:
I don't think this component should be called "Search" as this also contains a title. 
It's fine for now if you don't want to change it

Reply:
ok good, I will consider it for the future